src/plugins/render/datatypes/program.cpp

```diff
@@ -26,42 +26,76 @@ dependency_graph::State checkProgramState(const GLuint& programId) {
 
 }
 
-Program::Program() {
-	assert(glCreateProgram && "GLEW is probably not initialised");
+struct Program::Pimpl : public boost::noncopyable {
+	Pimpl(const std::vector<Shader>& shaders) : m_programId(0), m_shaders(shaders) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Struct 'Pimpl' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/manual/martin.prazak/possumwood/pullRequest?prid=4929755)